lib/Backend/BackwardPass.cpp

```diff
@@ -2295,29 +2310,23 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr, BailOutInfo * bailOutInfo)
             NEXT_BITSET_IN_SPARSEBV;
 
             // SIMD_JS
-            tempBv->Or(bailOutInfo->liveSimd128F4Syms, bailOutInfo->liveSimd128I4Syms);
-            tempBv->And(byteCodeUpwardExposedUsed);
-            byteCodeUpwardExposedUsed->Minus(tempBv);
-            FOREACH_BITSET_IN_SPARSEBV(symId, tempBv)
-            {
-                StackSym * stackSym = this->func->m_symTable->FindStackSym(symId);
-                Assert(stackSym->GetType() == TyVar);
-                StackSym * simd128Sym = nullptr;
-                if (bailOutInfo->liveSimd128F4Syms->Test(symId))
-                {
-                    simd128Sym = stackSym->GetSimd128F4EquivSym(nullptr);
-                }
-                else
-                {
-                    Assert(bailOutInfo->liveSimd128I4Syms->Test(symId));
-                    simd128Sym = stackSym->GetSimd128I4EquivSym(nullptr);
-                }
-                byteCodeUpwardExposedUsed->Set(simd128Sym->m_id);
-            }
+#define SIMD_CAPTURE_VALS(_TAG_) \
+            tempBv->And(byteCodeUpwardExposedUsed, bailOutInfo->liveSimd128##_TAG_##Syms); \
```

Comment:
I was planning to have this in a future PR. Will add it to this one instead then.